### ISSUE TITLE
[auth] call onError if user token already exists instead of return

### ISSUE THREAD

**Issue by shahidhk** [hasura/js-sdk#15]

**Comment by coco98** [comment]
@shahidhk Can you elaborate?

**Comment by shahidhk** [comment]
at [line 17 of Auth.js](https://github.com/hasura/js-sdk/blob/master/src/Auth.js#L17), 

```js
 if (this.hasura.user.token) {
      logError('A user session already exists. Use this.hasura.logout() first?');
      return;
}
```

If someone tried to login again, even if an `onError` callback is passed, it is not called. Hence this error cannot be captured from the client code. 

Client can check for existing sessions before showing the login screen, but `onError` callback should be able to catch these errors.

**Comment by coco98** [comment]
Fixed in v0.1.3
